Orange/widgets/data/owpaintdata.py

```diff
@@ -27,6 +24,9 @@
 from Orange.widgets.utils import itemmodels, colorpalette
 from Orange.widgets.io import FileFormat
 
+from Orange.util import scale, namegen
+import pyqtgraph as pg
```

Comment:
The module, as was, failed to run as "__main__" with latest pyqtgraph release. :(